packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/SelectedPersona.tsx

```diff
@@ -61,13 +60,10 @@ const SelectedPersonaInner = React.memo(
       dragDropEvents,
       eventsToRegister,
     } = props;
-    const itemId = getId();
+    const itemId = useId();
     const [root, setRoot] = React.useState<HTMLElement | undefined>();
```

Comment:
I think according to Elizabeth's comment this needed to get modified as well?

Reply:
yea i am working on that :) did not seem straightforward enough as it is being used in multiple places below..